.buildkite/scripts/e2e/util.py

```diff
@@ -0,0 +1,35 @@
+import docker
+import os
+import requests
+import subprocess
+from docker.models.containers import Container
+from requests.adapters import HTTPAdapter, Retry
+
+
+class Util:
```

Comment:
nit: you are already in a Python module called util, there's no need to add these functions as staticmethod under a class with the same name; they simply could be functions inside this module.

In that case, you'd change https://github.com/elastic/logstash-filter-elastic_integration/pull/122/files#diff-4948beae97f41385b2fa6d42111231d2f691ca2bef117bce62609a7cbee65b45R11 to `import util` and reference the functions like `util.get_home_path()` etc.

Reply:
Makes sense as it contains all static methods now, removed the class and applied the suggestion, thanks.